auto-man/Project.Rd

```diff
@@ -17,18 +17,20 @@ Projects in Synapse must be uniquely named. Trying to create a project with a na
 
 Attributes:
```

Comment:
Does the order of attributes in this documentation file cause issues if it's not followed? 

Specifically: If someone sees that `alias` is the second attribute does that cause an issue since the `alias` is really the last of them in the `usage` and `arguments` section.

Reply:
@BryanFauble I think the order matters. As my understanding, [man/Project.Rd](https://github.com/Sage-Bionetworks/synapser/blob/SYNR-1520-upgrade-python-client/man/Project.Rd) is the actual file to be knitted for the package doc. The attributes order matches what is specified in [Project object initiation section of the synapseclient](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/ea6c7a741ce36675a8ceb28de790b92a0f730b32/synapseclient/entity.py#L474). One thing I've noticed is the attributes order in the [docstring](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/ea6c7a741ce36675a8ceb28de790b92a0f730b32/synapseclient/entity.py#L455) does not match with what actually specified in the initiation section. Do we want to update it?

Reply:
Since we are moving to the new models eventually, I'm not sure it makes sense to update `Project` on the SYNPY side of things.

Reply:
Let's leave it as is. Since the man/Project.Rd reflect the actual order of attributes, I think we are good for now.